build.sbt

```diff
@@ -7,27 +7,27 @@ val V = new {
   val `cats-effect`      = "3.5.1"
   val circe              = "0.14.5"
   val ciris              = "3.2.0"
-  val confluent          = "7.2.2"
+  val confluent          = "7.4.1"
   val doobie             = "1.0.0-RC4"
+  val http4s             = "0.23.23"
   val jackson            = "2.15.2"
   val `jackson-databind` = "2.15.2"
   val `json-schema`      = "1.14.2"
   val `jsoniter-scala`   = "2.23.2"
   val kafka              = "3.4.1"
   val logback            = "1.4.8"
-  val `log-effect`       = "0.16.4"
+  val `log-effect`       = "0.17.0"
   val ociSdk             = "3.22.0"
   val postgres           = "42.6.0"
   val `scala-compat`     = "2.11.0"
   val slf4j              = "2.0.7"
-  val sttp               = "3.5.2"
-  val uzhttp             = "0.2.8"
```

Comment:
dropping this in favor of http4s as they never released a version compatible with zio 2